dev-libs/intel-neo/intel-neo-21.31.20514.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_PN="compute-runtime"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Intel Graphics Compute Runtime for L0 and OpenCL, for Broadwell and beyond"
+HOMEPAGE="https://github.com/intel/compute-runtime"
+SRC_URI="https://github.com/intel/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="l0 vaapi"
+
+BDEPEND="virtual/pkgconfig"
+COMMON=">=virtual/opencl-3
+	>=dev-util/intel-graphics-compiler-1.0.8173
+	>=media-libs/gmmlib-21.2.1
+	l0? ( >=dev-libs/level-zero-1.4.1 )
+	vaapi? (
+		x11-libs/libdrm[video_cards_intel]
+		x11-libs/libva
+	)"
+DEPEND="${COMMON}
+	media-libs/mesa"	# for Khronos OpenGL headers
+RDEPEND="${COMMON}"
```

Comment:
I'd use ${COMMON_DEPEND} for this if we had to, but let's just do RDEPEND first, then define DEPEND in terms of RDEPEND? (DEPEND="${RDEPEND}
	media-libs/mesa")

Nit: put BDEPEND last.

Reply:
Good point, these things were left unchanged from the previous version, but I am happy to clean it up now. Done